src/dotnet/commands/dotnet-clean/Program.cs

```diff
@@ -19,7 +19,10 @@ public CleanCommand(IEnumerable<string> msbuildArgs, string msbuildPath = null)
 
         public static CleanCommand FromArgs(string[] args, string msbuildPath = null)
         {
-            var msbuildArgs = new List<string>();
+            var msbuildArgs = new List<string>
```

Comment:
What was the default verbosity before?

Also, isn't this considered a breaking change, since it can potentially impact anyone who was relying on the previous default?